### PR TITLE
refactor(payments): update LoadingOverlay to utilize Tailwind

### PR DESCRIPTION
**Pull request by sardesam**
## Because

- We want to convert to Tailwind across fxa for consistency and to improve overall maintenance
- We want to improve responsiveness and UI/UX
- We want to improve accessibility

## This pull request

- Replace all CSS/SCSS with Tailwind utility classes
- Remove .css and .scss files
- Rework Storybook files as discussed in FXA-5080

Closes: #13797

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx [hunk 1]
```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
 
 export const LoadingOverlay = ({ isLoading }: LoadingOverlayProps) =>
   !isLoading ? null : (
```
Comment (xlisachan) [inline packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:12]: Nit: `isLoading && (`

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx [hunk 2]
```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
...
-    <div id="loading-overlay" data-testid="loading-overlay">
-      <img alt="" src={spinnerImage} />
+    <div
+      id="loading-overlay"
```
Comment (xlisachan) [inline packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:14]: Can we rip `id="loading-overlay"` out now?

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx [hunk 3]
```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
...
+      className="flex justify-center items-center fixed inset-0 z-[9999]"
```
Comment (xlisachan) [inline packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:15]: Left out Tailwind of `background: #fbfbfe`? Do we need it?

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx [hunk 4]
```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
...
+      data-testid="loading-overlay"
+    >
+      <img className="h-10 w-10 animate-spin" alt="" src={spinnerImage} />
```
Comment (xlisachan) [inline packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:18]: Aligned with updates.

**Review by xlisachan — APPROVED** [review]
r+wc! Just a nit and a q. :)

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx (hunk 2 above)
Comment (sardesam) [reply packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:14]: Hahaha yes! My bad. Forgot to remove it after deleting the stylesheet. 🙂 Thank you!

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx (hunk 3 above)
Comment (sardesam) [reply packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:15]: Nope. 👍

File: packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx (hunk 1 above)
Comment (sardesam) [reply packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx]: Nice. Updated!

**Pull request merged**